goal_src/jak2/levels/underport/underb-master.gc

```diff
@@ -84,8 +87,13 @@
           (s5-0 (new 'stack-no-clear 'vector))
           (gp-0 (new 'stack-no-clear 'vector))
           (f30-0 4096.0)
+          (ratio (* (-> *pc-settings* aspect-ratio) 0.375))
           )
-      (let ((f0-4 (* 0.00013563369 (tan (* 0.5 (-> *math-camera* fov))) f30-0)))
+      (set! *underb-ratio* ratio)
+      ;; clamp interp if aspect ratio is too wide
+      (set! (-> obj interp) (if (> ratio 1.0) 0.93 ratio))
```

Comment:
This goes from 0.99, to 1.0, then after that it clamps to 0.93? Confusing.

Reply:
Past 0.93, the effect gets a bit too subtle and completely stops at 1.0, so I just clamped it to this. I'll change the condition to `> ratio 0.93` for clarity.